### ISSUE TITLE
following password reset link results in argument type error

### ISSUE THREAD

**Issue by aaftre** [zacksmash/fortify-ui#33]
App\Providers\FortifyUIServiceProvider::App\Providers\{closure}(): Argument #1 ($request) must be of type App\Providers\Request, Illuminate\Http\Request given

```php
        Fortify::resetPasswordView(function (Request $request) {
            return view('auth.reset-password', ['request' => $request]);
        });

```

PHP 8.3.8
Laravel 11.2.0
"zacksmash/fortify-ui": "^2.0"

**Comment by zacksmash** [comment]
@aaftre Thanks for this! This is fixed in v2.0.1

**Comment by aaftre** [comment]
fixed by adding ```use Illuminate\Http\Request``` not sure if its in the install documentation or i missed it.

**Comment by aaftre** [comment]
> @aaftre Thanks for this! This is fixed in v2.0.1

great! thank you!
